project/src/main/java/com/todo/project/persistence/repository/TicketRepository.java

```diff
@@ -3,7 +3,20 @@
 import com.todo.project.persistence.model.Ticket;
 import org.springframework.data.jpa.repository.JpaRepository;
 
-public interface TicketRepository extends JpaRepository<Ticket, Long> {
```

Comment:
This way we lose the autogenerated methods for us by JPA, but we could include that in further PR.